src/utils/dayjs.ts

```diff
@@ -0,0 +1,16 @@
+import dayjs from 'dayjs';
+import timezone from 'dayjs/plugin/timezone';
+import utc from 'dayjs/plugin/utc';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+export function isValidTimeZone(tz: string) {
```

Comment:
can we memoize these results?

Reply:
Why should we memorize it ?
I'm not sure I understand what you mean 🤔

Reply:
So subsequent calls don't need to create a DateTimeFormat & Error object to know the result

Reply:
Sure 👍